robolectric/src/test/resources/res/values/typed_arrays.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <array name="typed_array_values">
+        <item>abcdefg</item>
+        <item>3875</item>
+        <item>2.0</item>
+        <item>#ffff00ff</item>
+        <item>#00ffff</item>
+    </array>
+    <array name="typed_array_references">
+        <item>@string/typed_array_a</item>
```

Comment:
If you need in the Java code we have constants for AttributeResource.{NULL_VALUE,EMPTY_VALUE} I think I also mentioned AttributeResource.isResourceReference(String value) + AttributeResource.isStyleReference(String value) - it might be a good idea to move these onto their own class, e.g: a new class AttributeValues or maybe even ResType if we need to use them outside of the creating an AttributeResource object